src/MailerMessage.php

```diff
@@ -321,36 +321,46 @@ public function setHtmlBody($html)
     }
 
     /**
-     * @inheritdoc
+     * Get Custom Campaign
+     * @see https://dev.mailjet.com/guides/?shell#group-into-a-campaign
+     * @since 1.1.3
      */
     public function getCustomCampaign()
     {
         return $this->_customCampaign;
     }
 
     /**
-     * @inheritdoc
+     * Set Custom Campaign
+     * @see https://dev.mailjet.com/guides/?shell#group-into-a-campaign
+     * @since 1.1.3
      */
     public function setCustomCampaign($campaign)
     {
         $this->_customCampaign = $campaign;
         return $this;
     }
  
-        /**
-     * @inheritdoc
+    /**
+     * Get the CustomID
+     * With the Custom ID you can group events, if the Custom ID is provided at send time
+     * @see https://dev.mailjet.com/guides/?shell#event-api-real-time-notifications
+     * @since 1.1.3
      */
-    public function getCustomID()
+    public function getCustomId()
     {
-        return $this->_customID;
+        return $this->_customId;
     }
 
     /**
-     * @inheritdoc
+     * Set the CustomID
+     * With the Custom ID you can group events, if the Custom ID is provided at send time
+     * @see https://dev.mailjet.com/guides/?shell#event-api-real-time-notifications
+     * @since 1.1.3
```

Comment:
Please also add the other phpdoc comments like @param string|int ??? (i assume int) and @return static

Reply:
Added Doc and Changelog.
If you miss something more, please let me know.